### PR TITLE
fix: use singleDocData

### PR DESCRIPTION
**Pull request by Crash--**
When refactored query() to fetchQueryAndGetFromState() we lose this option, we fixed the real issue in cozy-client https://github.com/cozy/cozy-client/commit/a840bf46e2d6b5f28572e3f1d9f9558765d8e085 but let's be explict, this is better

### CODE REVIEW

**Review by Merkur39 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "cozy-keys-lib": "^3.11.1",
     "cozy-realtime": "3.13.1",
     "cozy-sharing": "3.10.0",
+    "cozy-stack-client": "^33.2.0",
```
Comment (paultranvan) [inline package.json:65]: Why did you need to add this?
